### DISCUSSION TITLE
Dagre Cluster implementation example

### DISCUSSION THREAD

**Discussion by ROYopedia** [clientIO/joint#2210] [Q&A]
### Introduction

Hello, I was going through directed graph layout with jointjs. Dagre cluster is mentioned but no exmaple or anything I found. 

Could you please give one example implementation for dagre cluster?
![image](https://github.com/clientIO/joint/assets/22453886/2b2708a8-3ed4-44c3-ba30-af56fbc77729)

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by kumilingus** [comment]
Here's the source code for the demo from the documentation: https://resources.jointjs.com/docs/jointjs/v3.7/demo/layout/DirectedGraph/js/clusters.js

Here's a similar demo on JSFiddle: https://jsfiddle.net/kumilingus/yfbc5knv/
